### ISSUE TITLE
can you globally disable tracer via an environment variable?

### ISSUE THREAD

**Issue by zhammer** [DataDog/dd-trace-py#1279]
it'd be helpful for our local/test development environments if we could disable ddtrace globally via an environment variable.

i tried using `DD_TRACE_ANALYTICS_ENABLED` but that doesn't seem to do the trick
```
$ python -c "import ddtrace; print(ddtrace.tracer.enabled)"
True
$ DD_TRACE_ANALYTICS_ENABLED=false python -c "import ddtrace; print(ddtrace.tracer.enabled)"
True
```

also can't find any env var in the docs or from poking around the code here: https://github.com/DataDog/dd-trace-py/blob/3f8970e17e62fd81e916c14a0b5f5f70c2107036/ddtrace/tracer.py#L114

if it helps, our main requirement is that the traces aren't actually sent over network once done. it's still fine if the traces are instrumented locally!

**Comment by brettlangdon** [comment]
The only thing we have right now is `DATADOG_TRACE_ENABLED` but only available if you are using `ddtrace-run`.

https://github.com/DataDog/dd-trace-py/blob/3f8970e17e62fd81e916c14a0b5f5f70c2107036/ddtrace/commands/ddtrace_run.py#L27


It has been on my mental todo list to move this from `ddtrace-run` into `ddtrace/tracer.py`.

You could implement locally via something like:

```python
import ddtrace
import os

ddtrace.tracer.enabled = bool(os.getenv("DATADOG_TRACE_ENABLED") != "false")
```

(ish)

**Comment by brettlangdon** [comment]
I'll official add a card to our backlog for this.

**Comment by zhammer** [comment]
great. yeah in these cases we're not running via ddtrace-run.

can help out with a PR if needed

**Comment by brettlangdon** [comment]
cool, it _could_ be as easy as changing this line:

https://github.com/DataDog/dd-trace-py/blob/3f8970e17e62fd81e916c14a0b5f5f70c2107036/ddtrace/tracer.py#L114

```python
enabled=asbool(get_env("trace", "enabled', default=True))
```

**Comment by zhammer** [comment]
ha yeah that's what i was thinking :)

**Comment by maximillianus** [comment]
Ah sweet thread. I am also wondering how to disable tracing since `DATADOG_TRACE_ENABLED` env var does not seem being used in ddtrace library.

This info below not available in the latest version of ddtrace
https://github.com/DataDog/dd-trace-py/blob/3f8970e17e62fd81e916c14a0b5f5f70c2107036/ddtrace/commands/ddtrace_run.py#L27

This is the latest version:
https://github.com/DataDog/dd-trace-py/blob/a2d9f1bad616cbfcd0e82e5848dc0c9be62aa20f/ddtrace/commands/ddtrace_run.py#L18-L24

Adding additional code below as mention by @brettlangdon above would be great for user who doesn't use `ddtrace-run` like me 🙂: 
```python
enabled=asbool(get_env("trace", "enabled', default=True))
```

**Comment by brettlangdon** [comment]
This has been resolved by #1575. Apologizes for not closing this issue sooner.

https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/tracer.py#L143
